src/sass/components/_itemActionBar.scss

```diff
@@ -14,15 +14,23 @@
   margin: 1em 0;
   padding-top: 10px;
 
-  &__btn {
-    // currently uses Bootstrap .btn styling
+  &__btn-set {
     margin-right: 8px;
+  }
 
+  &__btn {
+    // currently uses Bootstrap .btn styling
     &--position-selected {
       color: $selected-color;
     }
   }
 
+  &__position-btn-label {
+    @include breakpoints (max mid-small) {
```

Comment:
This is a more appropriate breakpoint (and approach IMO) to hide position button (support/oppose) labels. @zachmonteith Utility classes are fine too (but I want to create and use them more intentionally - easily identifiable, reusable, consistent, and organized). We can chat more about this if you'd like.